features/create_book.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Create a new book
+    As an author
+    When I want to create a new book
+    I would like to fill a create_new_book form
+
+    Scenario: Creating a new book 
+    When I am on the Create New Book Page
```

Comment:
1. Indent code below the scenario
1. Scenario explains the scenario we are having for example 
- ` create a new book with all valid details`
- ` create a new book with invalid isbn`